src/views/Item20Items.js

```diff
@@ -0,0 +1,34 @@
+import kind from '@enact/core/kind';
```

Comment:
Sorry, I don't like the name of file.
I prefer "Items" or "ItemGroup" with datasize Prop